pkg/kubelet/cm/types.go

```diff
@@ -54,3 +54,10 @@ type CgroupManager interface {
 	// Update cgroup configuration.
 	Update(*CgroupConfig) error
 }
+
+// QOSContainersInfo hold the names of containers per qos
+type QOSContainersInfo struct {
```

Comment:
+1

On Fri, Jul 8, 2016 at 2:35 PM, Derek Carr <notifications@github.com> wrote:

> In pkg/kubelet/cm/types.go
> <https://github.com/kubernetes/kubernetes/pull/27853#discussion_r70144463>
> :
>
> > @@ -54,3 +54,10 @@ type CgroupManager interface {
> >  	// Update cgroup configuration.
> >  	Update(*CgroupConfig) error
> >  }
> > +
> > +// QOSContainersInfo hold the names of containers per qos
> > +type QOSContainersInfo struct {
>
> @vishh <https://github.com/vishh> - we should think about returning
> summary stats per qos tier in kubelet summary api
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27853/files/8134b9de63922520d329b09348c13f5cc110507e#r70144463>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGvIKLZsWf-mnBjVGJQIJJ8scuzqTv5Dks5qTsKMgaJpZM4I7o6F>
> .
>